semaphore/setup.sh

```diff
@@ -20,10 +20,10 @@ sudo swapon /swapfile
 
 # Ensure cache directories exist
 CACHE=$SEMAPHORE_CACHE_DIR
-mkdir -p "$CACHE/asdf/installs" "$CACHE/mix/deps" "$CACHE/npm"
+mkdir -p "$CACHE/asdf/installs" "$CACHE/mix" "$CACHE/npm"
 
 export MIX_ENV=test
-export MIX_DEPS_PATH=$CACHE/mix/deps
+export MIX_HOME=$CACHE/mix
```

Comment:
Has this changed because of analog changes on the Semaphore side?

Reply:
Without this, I ran into an issue compiling the Phoenix Node dependencies.